data-items.php

```diff
@@ -0,0 +1,29 @@
+<?php
+require_once 'form-functions.php';
+session_start();
+
+function getDBInsert() {
+    $db = new PDO('mysql:host=db; dbname=collector_project', 'root', 'password');
+    $db->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
+    $insert = "INSERT INTO `taxidermy_collection`(`name`,`origin`,`family`,`method`,`rating`,`image`,`description`) VALUES (:name,:origin,:family,:method,:rating,:image,:description);";
+    return $db->prepare($insert);
+}
+$data = gatherData($_POST);
+$insertItemArray = getDBInsert();
```

Comment:
if you passed the `$data` variable into the function, you could execute it inside the function instead of returning the prepared statement